src/pages/homepage/homepage.component.jsx

```diff
@@ -5,7 +5,7 @@ import CommunityList from "../../components/community-list/community-list.compon
 
 const HomePage = ({ currentUser }) => (
   <div>
-    <HeroText />
+    {!currentUser && <HeroText />}
```

Comment:
```suggestion
    {!currentUser ? <HeroText />}
```
Why not like that?

Personally I also believe some text should be shown and only the part where it mentions the login should be hidden.

Reply:
I agree with you that in future there might be some text that comes up when user logged in. I used && because it suits with current requirement as we don't have welcome text for the logged in user. I prefer not ?: ternary because the futuristic aspect of code. Once we know what text needs to render for a logged in user, we can go ahead and change

Reply:
Thanks for the explanation! :+1: